db/db_test2.cc

```diff
@@ -5581,6 +5650,7 @@ TEST_F(DBTest2, ChangePrefixExtractor) {
     // Re-execute similar queries after a full compaction
     {
       std::unique_ptr<Iterator> iterator(db_->NewIterator(ReadOptions()));
+      ASSERT_OK(iterator->status());
```

Comment:
Please move this either after the Valid() or before the iterator is deleted.

Reply:
Done.